src/main/java/roomescape/controller/api/admin/AdminReservationController.java

```diff
@@ -0,0 +1,65 @@
+package roomescape.controller.api.admin;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import roomescape.dto.request.AdminReservationRequest;
+import roomescape.dto.response.MultipleResponse;
+import roomescape.dto.response.ReservationResponse;
+import roomescape.service.ReservationService;
+
+import java.net.URI;
+import java.time.LocalDate;
+import java.util.List;
+
+@RequestMapping("/admin/reservations")
+@RestController
+public class AdminReservationController {
+
+    private final ReservationService reservationService;
+
+    public AdminReservationController(ReservationService reservationService) {
+        this.reservationService = reservationService;
+    }
+
+    @PostMapping
+    public ResponseEntity<ReservationResponse> addAdminReservation(@RequestBody AdminReservationRequest request) {
+        ReservationResponse reservationResponse = reservationService.addAdminReservation(request);
+
+        return ResponseEntity.created(URI.create("/reservations/" + reservationResponse.id()))
+                .body(reservationResponse);
+    }
+
+    @GetMapping
+    public ResponseEntity<MultipleResponse<ReservationResponse>> getAllReservations() {
+        List<ReservationResponse> reservations = reservationService.getAllReservations();
+        MultipleResponse<ReservationResponse> response = new MultipleResponse<>(reservations);
+
+        return ResponseEntity.ok()
+                .body(response);
+    }
+
+    @GetMapping("/filter")
+    public ResponseEntity<MultipleResponse<ReservationResponse>> getFilteredReservations(
+            @RequestParam(required = false) Long themeId,
+            @RequestParam(required = false) Long memberId,
+            @RequestParam(required = false) LocalDate dateFrom,
+            @RequestParam(required = false) LocalDate dateTo
+    ) {
+        List<ReservationResponse> reservations
+                = reservationService.getFilteredReservations(themeId, memberId, dateFrom, dateTo);
```

Comment:
저도 작성하면서 항상 하는 고민이라 크게 도움을 드리진 못하겠네요 😅
보통 인텔리제이 가이드선에 맞추고, 메서드 파라미터가 3개가 넘어가면 지금처럼 개행을 두는 편이긴 합니다~
하나 추가하자면 보통 변수 뒤에 `=`까지 붙이고 개행을 하는걸 추천드립니다!
```suggestion
        List<ReservationResponse> reservations = 
        reservationService.getFilteredReservations(themeId, memberId, dateFrom, dateTo);
```